apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -61,6 +61,11 @@ export const UnconnectedTeacherHomepage = ({
   const teacherReminders = useRef(null);
   const flashes = useRef(null);
 
+  /* We are hiding the AFE banner to free up space on the Teacher Homepage (September 2023) when
+   * we want to show the AFE banner again remove the next line and uses of 'shouldShowAFEBanner'.
```

Comment:
I know you are just copying the sentence structure from the other PR, but it sound a bit ambiguous so I am wondering if you can just make this simple change to it:
```suggestion
  /* We are hiding the AFE banner to free up space on the Teacher Homepage as of September 2023.
   * When we want to show the AFE banner again, remove the next line and uses of 'shouldShowAFEBanner'.
```